### ISSUE TITLE
Create Performance Tests

### ISSUE THREAD

**Issue by pjanotti** [dotnet/platform-compat#119]
As the number of cases covered increases for the PNSE analysis (e.g.: adding capability to detect field usage) we need to track performance impact on the analyzer. We also should be covering the case of larger projects.

@jmarolf any recommendation here? Follow CoreFX pattern? Go straight to Benchmark.NET?

**Comment by jmarolf** [comment]
@sharwell has some ideas I'm sure.  I assume the plan would be to eventually have these be part of some automated perf run?

**Comment by pjanotti** [comment]
Yes, that is correct - but initial goal is to have some manual suite to track performance impact.
